.github/ISSUE_TEMPLATE/hotfix-issue-release-template.yml

```
@@ -0,0 +1,172 @@
+    required: true
+
+- type: textarea
+  attributes:
+    label: OS Version
+    description: If you know what OS version the bug occurred on, enter it here.
+    placeholder: v1.2.3
+  validations:
+    required: false
+
+- type: textarea
+  attributes:
+    label: Anything Else
+    description: Links? References? Please provide anything that will give us more context about the issue you encountered. 
+    placeholder: "Tip: You can drag and drop files here (i.e. images, log files etc.)." 
+  validations:
+    required: false
+
+- type: checkboxes
+  attributes:
+    label: Pre-Release ToDo List
+    description: Complete the following items to perform a release.
+    options:
+    - label: Add `🔥Hot Fix Release🚀` label to this issue.  
+    - label: Issue is assigned to a project.
+    - label: Issue is assigned to a milestone.
+    - label: Release PR linked to this issue.
+    - label: All unit tests have been executed locally and have passed. _(Check out the appropriate release branch before running tests)_.
+    - label: Version in project file updated _(All changes made directly on the release branch (master))_.
+    - label: Release notes created and added _(All changes made directly on the release branch (master))_.
+    - label: Manual QA Testing completed _(if applicable)_.
+    - label: Pull request has been approved and merged into a release branch before performing the release. _(Releases are performed on the master branch)_.
+    - label: Release to **_production_** complete. _(The release is performed by running the `🚀Production Release` workflow)_.
+
+
+# POST-RELEASE TODO LIST
+- type: checkboxes
+  attributes:
+    label: Post-Release ToDo List
+    description: Verify that release went smoothly.
+    options:
+    - label: The GitHub release has been created and is correct. 
+    - label: The NuGet package has been successfully deployed to [nuget.org](https://www.nuget.org/) _(if applicable)_.
+    - label: Announcement of release on [Twitter](https://twitter.com/KDCoder) verified _(if applicable)_. _(Announcement should be performed automatically with the release)_.
+    - label: Announcement has been pushed to the [Discord](https://discord.gg/qewu6fNgv7) channel _(if applicable)_.
+
+- type: textarea
+  attributes:
+    label: "Additional Information:"
+    description: DO NOT CHANGE ANYTHING BELOW!!
+    value: |
+      **_<details closed><summary>HotFix Purpose</summary>_**
+
+      The purpose of a hotfix is to release a fix to something wrong to production as soon as possible.
+      This concept should be treated very carefully.  This should only happen if the software is simply broken and
+      _**UNUSABLE**_.  If there is a simple workaround or the issue is minor, then a hotfix should not be required.
+
+      A hotfix should be a rare occasion and if it has been occurring often, then the development and release process
+      should be reviewed for improvements to try and help prevent a hotfix from occurring.
```

Reword  - Do you need all of this since it's pretty much described above? 
```suggestion
      If a hotfix is recurring, then the development and release process
      should be reviewed for improvements to prevent a hotfix from happening.
```